spec/dummy/spec/system/csv_import_details_page_spec.rb

```diff
@@ -117,12 +117,16 @@
     expect(page).to have_content('user@curationexperts.com')
   end
 
-  xit 'has pagination for PreIngestWorks at 10' do
+  it 'has pagination for PreIngestWorks at 10' do
     visit('/csv_import_details/index')
+    sleep(2)
     click_on 'Next'
+    sleep(2)
```

Comment:
Are we sure that the click_on method needs an explicit wait after it?  These two articles make it appear that only the visit method has the problem - see "Dangerous Capybara Wait Matchers" here https://engineering.gusto.com/eliminating-flaky-ruby-tests/ - and "Capybara to the Rescue" here https://thoughtbot.com/blog/write-reliable-asynchronous-integration-tests-with-capybara

I think there might be a version of this test that looks like
```
visit('/csv_import_details/index')
find("Next", match: first).click   # match first is only required if we're worried about multiple next buttons
find("4").click                    # if we're not worried about multiple clickable links with the label '4' on the page
find('Next').click                 # combines sleep, expect, and click steps in a way that uses Capybara's in-buit waiting
expect(page).to have_content 'Previous'
```